tests/testthat/test-methods.R

```diff
@@ -22,27 +22,34 @@ test_that("it returns the input trimmed to the last full lint if one exists with
 test_that("as.data.frame.lints", {
   # A minimum lint
   expect_is(
-    l1 <- Lint("dummy.R",
-         line_number = 1L,
-         type = "style",
-         message = "",
-         line = ""),
+    l1 <- Lint(
+      "dummy.R",
+      line_number = 1L,
+      type = "style",
+      message = "",
+      line = ""
+    ),
     "lint"
   )
 
   # A larger lint
   expect_is(
-    l2 <- Lint("dummy.R",
-              line_number = 2L,
-              column_number = 6L,
-              type = "error",
-              message = "Under no circumstances is the use of foobar allowed.",
-              line = "a <- 1",
-              ranges = list(c(1, 2), c(10, 20)),
-              linter = "custom_linter"),
+    l2 <- Lint(
+      "dummy.R",
+      line_number = 2L,
+      column_number = 6L,
+      type = "error",
+      message = "Under no circumstances is the use of foobar allowed.",
+      line = "a <- 1",
+      ranges = list(c(1, 2), c(10, 20))),
     "lint"
   )
 
+  expect_warning(
+    Lint("dummy.R", linter = "deprecated"),
+    fixed = "deprecated"
```

Comment:
isn't `fixed` the argument passed to `grepl`?

Reply:
Yep, the warning should contain the text "deprecated", no?

Reply:
then fixed=TRUE and I think warning="deprecated" if you want to name it right?

Reply:
Oh, thanks for pointing that out.
We have a lot of these calls in `test-lint_file.R`